lib/commands/upgrade.js

```diff
@@ -49,8 +51,8 @@ module.exports = function () {
   function upgradeKick() {
     Logger.log('Updating kick...'.white);
 
-    child_process.execSync('npm install -g kick');
```

Comment:
it was later changed to Utils.spawnProcess('npm', ['install', '-g', 'kick'], null, { sync: true });